src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -351,7 +351,7 @@ class ActivityFeed extends React.Component<Props, State> {
             // eslint-disable-next-line jsx-a11y/no-static-element-interactions
             <div
                 className={classNames('bcs-activity-feed', { 'bcs-is-scrolled': isScrolled })}
-                data-testid="activityfeed"
+                data-testid={isScrolled ? 'activityfeedscrolled' : 'activityfeed'}
```

Comment:
Should this come with a test of using the test id?

Reply:
Good idea @JChan106! I've added unit test to cover test id usage 👍